pkg/cstorpoolcluster/v1alpha1/cstorpoolspecs/v1alpha1/build.go

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright 2019 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	apisv1alpha1 "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	raidgroup "github.com/openebs/maya/pkg/cstorpoolcluster/v1alpha1/raidgroups/v1alpha1"
+	"github.com/pkg/errors"
+)
+
+// Builder is the builder object for PS
+type Builder struct {
+	ps   *PS
```

Comment:
what is PS? Can it be `CSPSpecs`. I am not sure if I am making any sense here.
Perhaps `PoolSpecs`.

If we have a valid custom resource with a valid alias, we can use aliases. Otherwise, aliases are difficult to communicate across teams.